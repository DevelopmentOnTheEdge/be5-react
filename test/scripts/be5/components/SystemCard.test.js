import React from 'react';
import renderer from 'react-test-renderer';
import '../../../../src/scripts/be5/be5init'
import {TestProvider} from "../testUtils";
import {Document} from "../../../../src/scripts/be5/containers/Document";
import be5 from "../../../../src/scripts/be5/be5";
import {MAIN_DOCUMENT} from "../../../../src/scripts/be5/constants";


test('snapshot', () => {
  const component = renderer.create(
    <TestProvider>
      <Document frontendParams={{documentName: MAIN_DOCUMENT}}/>
    </TestProvider>
  );

  be5.url.process({documentName: MAIN_DOCUMENT}, "#!systemCard" );
  expect(component.toJSON()).toMatchSnapshot();
});
