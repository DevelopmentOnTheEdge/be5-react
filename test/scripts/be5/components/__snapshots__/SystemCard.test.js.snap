// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div
  className="document-content"
  id="document-content___MAIN_DOCUMENT"
>
  <div>
    <div
      className="info-card"
    >
      <h1
        style={
          Object {
            "marginBottom": "13px",
          }
        }
      >
        System card
      </h1>
      <div
        className="navs-component"
      >
        <ul
          className="nav nav-tabs"
        >
          <li
            className="nav-item"
          >
            <a
              className="nav-link active"
              href="#!table/_system_/Cache"
              onClick={[Function]}
            >
              Cache
            </a>
          </li>
          <li
            className="nav-item"
          >
            <a
              className="nav-link"
              href="#!table/_system_/Session variables"
              onClick={[Function]}
            >
              Session
            </a>
          </li>
          <li
            className="nav-item"
          >
            <a
              className="nav-link"
              href="#!table/systemSettings/All%20records"
              onClick={[Function]}
            >
              System Settings
            </a>
          </li>
          <li
            className="nav-item"
          >
            <a
              className="nav-link"
              href="#!table/_system_/Daemons"
              onClick={[Function]}
            >
              Daemons
            </a>
          </li>
          <li
            className="nav-item"
          >
            <a
              className="nav-link"
              href="#!table/_system_/DataSource"
              onClick={[Function]}
            >
              DataSource
            </a>
          </li>
          <li
            className="nav-item"
          >
            <a
              className="nav-link"
              href="#!table/_system_/Http Headers"
              onClick={[Function]}
            >
              Http Headers
            </a>
          </li>
          <li
            className="nav-item"
          >
            <a
              className="nav-link"
              href="#!table/_system_/System properties"
              onClick={[Function]}
            >
              Properties
            </a>
          </li>
          <li
            className="nav-item"
          >
            <a
              className="nav-link"
              href="#!table/_system_/Threads"
              onClick={[Function]}
            >
              Threads
            </a>
          </li>
          <li
            className="nav-item"
          >
            <a
              className="nav-link"
              href="#!uiPanel"
              onClick={[Function]}
            >
              UI panel
            </a>
          </li>
        </ul>
        <div
          className="tab-content"
        />
      </div>
    </div>
  </div>
</div>
`;
